Kitodo/pom.xml

```diff
@@ -49,9 +49,9 @@
             <version>3.0.0-beta.2-SNAPSHOT</version>
         </dependency>
         <dependency>
-            <groupId>org.kitodo</groupId>
-            <artifactId>kitodo-ugh</artifactId>
-            <version>3.0-SNAPSHOT</version>
+            <groupId>jdom</groupId>
```

Comment:
why did you add jdom here?

Reply:
See description above: It was previously transitively added through the dependencies of the UGH. Removing the UGH requires to add it here manually for the project to compile.